pages/board.vue

```diff
@@ -1,63 +1,163 @@
+                    twitterHandle: "Protagorist",
+                    linkedInAccount: "",
+                    slackAccount: "Ian Phillipchuck",
+                },
+                {
+                    id: 3,
+                    name: "Lauren Briske",
+                    title: "Treasurer",
+                    imageLink: require("~/assets/boardPictures/laurenBriske.jpg"),
+                    description:
+                        "Lauren Briske is the Interim Director at <a class='text-indigo-600' href='https://startupedmonton.com/'>Startup Edmonton</a>. She is a passionate supporter of the startup and tech community and knows that a thriving tech ecosystem grows on the shoulders of a strong community.<br /><br />Through the Dev Edmonton Society, her goal is to empower local leaders to contribute to a growing community and to welcome more students, experienced developers, and aspiring developers into this great community.<br /><br />When she’s not working or volunteering, she’s probably riding down a mountain somewhere on skis or bikes.",
+                    twitterHandle: "",
+                    linkedInAccount:
+                        "https://www.linkedin.com/in/lauren-briske-03ba902b/",
+                    slackAccount: "laurenbriske",
+                },
+                {
+                    id: 4,
+                    name: "Mandy Meindersma",
+                    title: "Secretary",
+                    imageLink: require("~/assets/boardPictures/mandyMeindersma.jpg"),
+                    description:
+                        "Mandy Meindersma is a Backend Developer at <a class='text-indigo-600' href='https://dotdash.com/'>Dotdash</a>. She loves web development, teaching kidlets about computing science and hardware (making robots, light up costumes and IoT type of things).<br /><br /> She has loved getting more involved with DES through YEGWIT (Edmonton Women In Tech), CODEVID-19 (COVID-19 Hackathon) and now being able to help with organization and memberships!<br /><br /> Outside of tech she absolutely loves acro yoga, her cat Goose, and writing letters.",
+                    twitterHandle: "",
+                    linkedInAccount:
+                        "https://www.linkedin.com/in/mandy-meindersma/",
+                    slackAccount: "Mandy Meindersma",
+                },
+                {
+                    id: 5,
+                    name: "Abram Hindle",
+                    title: "Member at large",
```

Comment:
Can you change this to "Member At Large". I feel like it makes them seem less important which isn't the case. 😛 

```suggestion
                    title: "Member At Large",
```

Reply:
Haha for sure